src/src/AppBundle/Controller/DefaultController.php

```diff
@@ -53,13 +84,61 @@ public function indexAction($slug, $_locale, Request $request)
         $context['lang'] = $languageContext;
 
         if ($form->isSubmitted() && $form->isValid()) {
-            $context['attendee'] = $attendee;
+            $email = $form->getData()['email'];
+            $firstName = $form->getData()['first_name'];
+            $lastName = $form->getData()['last_name'];
+
+            // TODO - porovnavat na attendee,event
+            $attendeeExists = $em->getRepository(Attendee::class)->findOneBy(array('email' => $email));
+            if ($attendeeExists) {
+                // attendee already registered for event
+                // todo: Viki: problem je v tom, ze view si uz vytvorila a teda sa tato chyba vo view neukaze.
+                $form->get('email')->addError(new FormError('Attendee already exists.'));
+                $context['form'] = $form->createView();
+            } else {
+
+                $event = $em->getRepository(Event::class)->find($form->getData()['event_choice']);
+                $registration->setEvents($event);
 
-            $this->sendEmail($attendee, $context, $templateName, 'registration');
+                //todo check capacity
+                $countOfRegistratedPeople = $em->getRepository(Registration::class)->findBy(['events'=> $event]);
```

Comment:
uprimne, netusim ako toto funguje, nieje to nahodou array alebo nieco iterable? ci to sa smie v PHPcku porovnavat?